scripts/evergreen/e2e/single_e2e.sh

```diff
@@ -27,6 +27,21 @@ deploy_test_app() {
       tag="${OVERRIDE_VERSION_ID}"
     fi
 
+    local arch
+    arch=$(uname -m)
+
+    case "${arch}" in
+        ppc64le)
+            tag="${tag}-ppc64le"
+            ;;
+        s390x)
+            tag="${tag}-s390x"
+            ;;
+        *)
+            echo "Not IBM host, using default tag"
+            ;;
```

Comment:
q: do we HAVE TO add arch as suffix in the tag? Ideally we should not do this right?